config/menus/glue.cfg

```diff
@@ -24,6 +24,20 @@ ui_right = [ ui_list [ ui_spring 1; arg1 ] ]
 ui_center = [ ui_list [ ui_spring 1; arg1; ui_spring 1 ] ]
 ui_center_z = [ ui_list [ ui_spring 1; ui_list [ arg1 ]; ui_spring 1 ] ]
 
+
+ui_border_box = [
+    ui_list [
+        ui_background $ui_color_textfield_background $ui_blend_textfield_background $ui_color_textfield_border $ui_blend_textfield_border 1
+        ui_strut $arg1
+        ui_list [
+            ui_strut $arg3
+            arg5
+            ui_strut $arg4
+        ]
+        ui_strut $arg2
+    ]
+]
+
```

Comment:
This needs documenting comments describing what it is and how to use it.

Reply:
oh, yeah, ok